LICENSE.md

```diff
@@ -199,3 +202,7 @@
    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
    See the License for the specific language governing permissions and
    limitations under the License.
+
+##Content
+
+The Creative Commons Atribution-ShareAlike 4.0 license applies to the content inside of /website/content and /docs directory.
```

Comment:
please add full license text

```
The link:https://creativecommons.org/licenses/by-sa/4.0/[Creative Commons
Atribution-ShareAlike 4.0] license applies, unless otherwise stated to the
content created as part of this website, such as the Asciidoc or Markdown files
inside of `content/`.

image::https://licensebuttons.net/l/by-sa/4.0/88x31.png[Attribution/Share-a-like]

* link:https://creativecommons.org/licenses/by-sa/4.0/[Human-readable license text]
* link:https://creativecommons.org/licenses/by-sa/4.0/legalcode[Full legal license text]
```